### ISSUE TITLE
Repeater Item label + (most) controls hidden when inside nested Repeater Matrix Item

### ISSUE THREAD

**Issue by Rayden** [processwire/processwire-issues#293]
### Short description of the issue

I have a Repeater Item Field assigned to a Repeater Matrix Item, as soon i have a nested Repeater Matrix Item, the label of the Repeater Item will not be shown anymore (it only shows the label as ... and the control to expand/collapse). 

This only occurs when the Repeater Matrix Item is loaded as closed and opened by clicking on the Label. 

When 'Remember which repeater items are open' is checked, the Repeater Matrix Item is open, and the page is reloaded. The Repeater Item Label and controls just show as expected.

### Expected behavior

I would expect the Repeater Item Label to be shown, even when the Repeater Matrix Item containing it is opened and its content loaded by ajax.

### Actual behavior

The Repeater Item Label is set to 

### Optional: Screenshots/Links that demonstrate the issue

![screen shot 2017-06-16 at 09 07 25](https://user-images.githubusercontent.com/364556/27215836-684b302c-5273-11e7-924c-291bb90ed690.png)

### Steps to reproduce the issue

1. Create a Repeater Matrix Field and set **Item depth** to 1 and check **Remember which repeater items are open**
2. Create a Repeater Field
3. Add two Items to the Repeater Matrix Field
4. Assign the Repeater Field to one of the Repeater Matrix Items
5. Assign the Repeater Matrix Field to a Template
6. Create a Page with this Template
7. Add a Repeater Matrix Item
8. Add another Repeater Matrix Item (containing the Repeater Field) and drag it to be a sub-item of the first item
9. Keep all the Repeater Matrix Items opened and Save the Page
10. Return to the Page, now the Repeater Field Item Label and Controls are showing just fine.
11. Close all the Repeater Matrix Items and Save the Page
12. Return to the Page, and open the nested Repeater Matrix Item, now the Repeater Field Item Label and Controls are hidden

### Setup/Environment
* ProcessWire 3.0.62
* FieldtypeRepeaterMatrix 0.0.3
* FieldtypeRepeater 1.0.6
* PHP 5.5.8
* MySQL mysqlnd 5.0.11-dev

**Comment by ryancramerdesign** [comment]
Thanks for the report. It looks like there is a lot to setup here to see the issue. Before I set this up, first wanted to check if it might be possible to either export the profile, or send me the SQL dump used by the site that is seeing this issue? (email ryan processwire.com)

**Comment by Toutouwai** [comment]
Possibly related to this issue: https://github.com/processwire/processwire-issues/issues/173

**Comment by Rayden** [comment]
Thanks for looking in to this issue. @ryancramerdesign I send you an email with a link to the files of the project and database dump.

**Comment by Rayden** [comment]
This might also be related: I noticed when having a MapMarker field inside a matrix item, the map will not be rendered when opening a matrix item containing a Google map. Altough reloading the page when the matix item is opened it will render the map correctly.

Is there anyway i can provide some more info or data or help in any other way?

**Comment by ryancramerdesign** [comment]
Sorry this took awhile, I never could get PhpMyAdmin to import the DB, so had a couple of false starts and came back to it later. Finally just did it with the mysql client and everything worked. The fix turned out to be simple, changing a "filter" to a "find" in the javascript. Thanks for your help in tracking this one down.

I'm not sure if the MapMarker issue you observed is related or not. It's worth a try to see if this update fixes it. It's also possible that MapMarker needs an update to support dynamic loading, though if I recall correctly it should already support that.

**Comment by adrianbj** [comment]
There are a lot of mentions (issues and PRs) related to ajax and hidden field loading of MapMarker in it's GH repo: https://github.com/ryancramerdesign/FieldtypeMapMarker/ which may be related to this.

**Comment by Rayden** [comment]
Thanks Ryan for solving this issue, i will close this issue.
